### PR TITLE
Add babel ES6 transpiling

### PR DESCRIPTION
**Pull request by Serubin**
Currently, building to production will not work with vue-h-ajex as this module does not transpile from ES6 automatically causing an issue when Uglyify attempts to minify the vendors file.

This PR adds transpiling to the build process.

### CODE REVIEW

**Comment by zont** [comment]
What about Browserify and Webpack?

**Comment by Serubin** [comment]
Should work for both. Adding the build script takes care of all of it. And the prepare script will build before it hits the NPM repo.

File: package.json [hunk 1]
```diff
@@ -27,16 +30,16 @@
...
     "babel-preset-es2015": "^6.22.0",
+    "babel-preset-es2015-rollup": "^3.0.0",
     "eslint": "^3.15.0",
     "jest": "^18.1.0",
+    "rollup": "^0.41.1",
+    "rollup-plugin-babel": "^2.7.1",
+    "rollup-plugin-node-resolve": "^2.0.0",
     "vue": "^2.1.10"
   },
-  "browserify": {
```
Comment (zont) [inline package.json]: You deleted this section. It need for browserify without any prebuid...

Comment (zont) [reply package.json]: >  this module does not transpile from ES6 automatically
This section enable Browserify transpile from ES6 automatically

**Comment by zont** [comment]
May be Rollup has config options (like Browserify) to automatic compile dependencies?

**Comment by Serubin** [comment]
That's what [rollup.config.js](https://github.com/zont/vue-h-ajax/pull/1/files#diff-ff6e5f22a9c7e66987b19c0199636480) does.

When installed using `npm i vue-h-ajax` it will compile everything. The package.json specifies dist/index.js to be main file, which should get used when packaging dependencies.

My only issue is that it will not automatically compile when using just `npm i` and installing as a dependency. My understanding is that this is solved by pre-compiling it when uploading it to npm using prepare.

**Comment by zont** [comment]
You solve your problem, but broke compatibility with Browserify

**Comment by Serubin** [comment]
e4b7459 should fix the issue. 

My assumption was that rollup would take care of broswerify - I was mistaken. Let me know if this works.

**Comment by zont** [comment]
What about Jest tests run?
Why script deleted?

**Comment by zont** [comment]
Try to link you module (git://github.com/Serubin/vue-h-ajax.git) to browserify project:
image

**Comment by Serubin** [comment]
The error appears to be due to a missing dependency. babel-plugin-external-helpers

**Comment by Serubin** [comment]
I'll look into adding browserify support - I'll create a new pull request when that is working.

**Pull request closed without merging**